staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -554,17 +554,17 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			versionedObject = defaultVersionedObject
 		}
 		reqScope.Namer = action.Namer
-		namespaced := ""
+		namespaced := "Cluster"
```

Comment:
@smarterclayton It's because of this change I made (in order to be consistent with our naming scheme you suggested).